browser/tor/brave_tor_browsertest.cc

```diff
@@ -348,6 +350,61 @@ IN_PROC_BROWSER_TEST_F(BraveTorTest, ResetBridges) {
   EXPECT_FALSE(CheckComponentExists(tor::kTorPluggableTransportComponentId));
 }
 
+IN_PROC_BROWSER_TEST_F(BraveTorTest, HttpAllowlistIsolation) {
+  // Normal window
+  Profile* main_profile = browser()->profile();
+  auto* main_storage_partition = main_profile->GetDefaultStoragePartition();
+  content::SSLHostStateDelegate* main_state =
+      main_profile->GetSSLHostStateDelegate();
+
+  // Incognito window
+  Browser* incognito_browser = Browser::Create(Browser::CreateParams(
+      browser()->profile()->GetPrimaryOTRProfile(/*create_if_needed=*/true),
+      true));
```

Comment:
we have this: `InProcessBrowserTest::CreateIncognitoBrowser`

so this can just be replaced with `Browser* incognito_browser = CreateIncognitoBrowser(nullptr);`

Reply:
fixed